_posts/2021-04-25-ocp-java11.md

```diff
@@ -0,0 +1,290 @@
+Since each question weights as much as others, just risk that you'll not answer those and get points on others.
+You'll need to decide this fast!
+Of course, you should do this only for few questions.
+I had about 4. Already planned for skipping was nested labeled loops where you need to answer where to insert `break <LABEL>` or `continue <LABEL>` to get a specific output.
+I had such a question and immediately skipped it.
+The time is well spent on others.
+
+Overall the questions were not easy and often tricky, but you know that from practicing the questions in the Study Guide and the mock exams.
+I had the feeling that the mock exam questions were a bit harder than the questions I had in the exam.
+So if you are doing well on those tests, you are well prepared!
+
+There were some questions of course that cause compilation errors.
+As recommended follow the call sequence and ask yourself "Does it compile?".
+I spotted some errors like datatype conversion, extending/overriding final classes / methods, and reducing visibility of overridden methods.
+Could be that I missed some, at the end I made of course some mistakes.
+You'll get at the end the topic areas where you made mistakes but do not see the questions again.
+
+I had a moderate paste in going through the questions.
+Each question I read with care and also the answers.
+After doing this I was often clear which options I would answer.
+Sometimes I was not completely sure and marked those questions for review later when I was done with all.
+
+At the end I had less about 10 minutes for the skipped and marked questions.
+I needed that time to answer the skipped ones, and no time was left to review unsure ones.
+For the mentioned nested loop question I just did a wild guess and it surely was wrong.
+
+After the time was over the exam finished automatically and you get an immediate answer if you passed and the correct answer percentage.
+I was glad that I passed and this was all over now.
+
+# Conclusion
+
+Doing this exam is an intense experience and requires well preparation, even for experienced Java developers.
+When you are experienced take this as a chance to learn details and challenge yourself.
+Hopefully you are not doing the exam because you need just the badge in your CV and just want to pass somehow, but really learn and improve your skills.
+
+I recommend studying the OCP Java 11 Complete Study Guide in detail, and additionally practicing with the Mock Exams from Enthuware.
+If you do this you are really well prepared for taking the exam!
+However, do not underestimate the effort for doing so.
+Wouldn't I have sacrificed some of the vacation and spare time, I would not have made it through the study material.
+I might have passed, but when I would have before studying the whole material, I won't have touched them again.
+Not all of our study group made it through the material because they did not have enough time besides their regular work.
```

Comment:
```suggestion
Not everyone in the study group made it through the material in time.
Mainly because they did not have enough time besides their regular work.
```